src/components/organisms/HelpCenter/HelpCenter.tsx

```diff
@@ -25,18 +27,32 @@ export const HelpCenter: React.FC = () => {
     [startLoading]
   );
 
+  const handleBackClicked = () => {
+    setUrl("");
+  };
+
```

Comment:
Isn't there a better way, like a boolean to toggle `true`/`false` if the frame should be displayed at all without changing the URL?

Something like 
```tsx
url && isShown ?
```
☝️ 
this can use a hook like
```tsx
  const [isShown , toggleShown] = useToggle(false);
```

Reply:
@goran-peoski-work fixed in d544e65e1fe2d1354769ff619730270cb4d4b5d7